### ISSUE TITLE
Pairing Only Works With `Jurisdiction Tax Lot ID`

### ISSUE THREAD

**Issue by axelstudios** [SEED-platform/seed#4117]
**Describe the bug**
Importing records with both Property and Tax Lot columns will only pair if one of the Tax Lot columns is `Jurisdiction Tax Lot ID`.  Otherwise, the records will lose their relationship.

**Expected Behavior**
A single row in an import file with both property and tax lot columns should result in a paired record

**Actual Behavior**
Pairing only happens when `Jurisdiction Tax Lot ID` is mapped

**Steps to Reproduce**
Import the following csv files, mapping the columns to the `Property` and `Tax Lot` inventory types respectively:

1. ✅ The property and taxlot will pair as expected:
    | **PM Property ID** | **Jurisdiction Tax Lot ID** |
    |---|---|
    | P1 | J1 |

2. ❌ The property and taxlot will **NOT** be paired:
    | **PM Property ID** | **Custom ID 1** |
    |---|---|
    | P2 | C2 |

3. ❌ The property and taxlot will **NOT** be paired:
    | **PM Property ID** | **Address Line 1** |
    |---|---|
    | P3 | 3 Main St |

Or, simply import `covered-buildings-sample.csv` and map using the Tax Lot field `Custom ID 1` instead of `Jurisdiction Tax Lot ID`.

**Comment by RDmitchell** [comment]
@axelstudios -- I guess in this case the Custom ID 1 and Address Line 1 were fields that were assigned to the Tax Lot table, but then were not paired with the record in the Property table? Just wanted to make sure I understood the issue (!)

**Comment by axelstudios** [comment]
@RDmitchell Right - but after talking with @nllong this is working as expected.  Apparently we only pair records on import, even if they come from the same spreadsheet row, if `Jurisdiction Tax Lot ID` is one of the mapped columns.  It feels a little unintuitive to me, but this isn't technically a bug or regression so I'm going to change it to a feature request

**Comment by haneslinger** [comment]
@kflemin This is a feature request, should it be on this board?

**Comment by kflemin** [comment]
@nllong, thoughts on this?

**Comment by nllong** [comment]
Yeah, this is by design. If we want to make pairing more flexible, then we should ask for funding to do it. It is a complicated task and probably should happen outside of the "bug fix" quarter.

**Comment by RDmitchell** [comment]
We might want to query BEAM and see if they have come across the need for this pairing in their work with various jurisdictions. 

I think it would probably be a good idea to allow more pairings, but these days, not that many jurisdictions are using Tax Assessor data, and if they are, they will have some sort of ID from that data. 

So I would say lets see if the use case exists in the user base (or potential user base) before deciding to implement this.

**Comment by nllong** [comment]
if there is a consensus to enable more flexible pairings, then let's do it! Yes, let's ask BEAM.

**Comment by kflemin** [comment]
Better documentation would solve this issue. Not fixing for now.

Document this on the pairing page.
